### ISSUE TITLE
Feature requests

### ISSUE THREAD

**Issue by hazarek** [sbarex/SourceCodeSyntaxHighlight#57]
Feature requests;
- Apply or save without quit app
- pasting hex colors to color editor.
- importing .theme files
- enable/disable source file format.
- Installation with Homebrew cask.

**Comment by hazarek** [comment]
@sbarex If you open the discussion page, I would like to talk about the problems I experienced regarding the interface.

**Comment by sbarex** [comment]
I have enabled the discussion page.

On future release I will implement these request:
- Apply or save without quit app
- Importing .theme files

For "pasting hex colors to color editor", I don't think i will implement it, the interface is already pretty full so without adding other stuff. Some time ago I developed some plugins for the standard macOS color picker (which therefore can be used by all applications that use the standard color picker) that also handle the input of hexadecimal values. Maybe I'll consider making them available in new repositories.

In not possible to "enable/disable source file format". Apple do not allow this option.

Already exists an old version on Homebrew cask but I do not kwnow how update with the last relesase.
